test/unit/config-processor.test.js

```diff
@@ -1,6 +1,10 @@
-const assume = require('assume');
-const sinon = require('sinon');
-const processConfig = require('../../config-processor');
+import assume from 'assume';
+import assumeSinon from 'assume-sinon';
+import sinon from 'sinon';
+
+assume.use(assumeSinon);
+
+import processConfig, { applyExcludeList, applyIncludeList } from '../../config-processor.js';
```

Comment:
IMHO stylistically, imports belong above code.

Also IMHO move the `assume.use(assumeSinon)` into a global test setup so you don't have to do it everywhere or assume some file executed before another (i.e. this is leaky)

Reply:
I like to do this only in the test files that use `sinon` as the typings for `assume-sinon` extend the `assume` types in VS Code, so the `import` actually does something.